<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WSystemPara</class>
 <widget class="QWidget" name="WSystemPara">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>-100</y>
     <width>511</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>系统参数</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>250</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>应用</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>读取</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>232</width>
      <height>164</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>阶段配时表数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_7">
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>检测器：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_8">
         <property name="maximum">
          <number>48</number>
         </property>
         <property name="value">
          <number>32</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>启动黄闪时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_9"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>启动红灯时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_10"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_11">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>黄闪频率：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_11"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_12">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>信号机时间：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEdit"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>231</width>
      <height>164</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>通道数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox">
         <property name="maximum">
          <number>24</number>
         </property>
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>普通相位数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_2">
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>跟随相位数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_3">
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="value">
          <number>8</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>时基表数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_4">
         <property name="maximum">
          <number>40</number>
         </property>
         <property name="value">
          <number>40</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>时段表：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_5">
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>方案表数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="spinBox_6">
         <property name="maximum">
          <number>16</number>
         </property>
         <property name="value">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="bcBtn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>80</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>广播</string>
   </property>
  </widget>
  <widget class="QTableView" name="tbViewTscIp">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>331</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>521</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QPushButton" name="btnRead">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>录入</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSave">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>230</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>保存</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>60</y>
      <width>97</width>
      <height>172</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="cbModule">
       <property name="text">
        <string>模块表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbPhase">
       <property name="text">
        <string>相位灯组界面</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbConflict">
       <property name="text">
        <string>冲突相位表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbParttern">
       <property name="text">
        <string>方案表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbStage">
       <property name="text">
        <string>阶段配时表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbDet">
       <property name="text">
        <string>检测器表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbTimeBase">
       <property name="text">
        <string>时基表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSchedule">
       <property name="text">
        <string>时段表</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="combRead">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="combSave">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>69</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>97</width>
      <height>172</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_6">
     <item>
      <widget class="QCheckBox" name="cbModule_2">
       <property name="text">
        <string>模块表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbPhase_2">
       <property name="text">
        <string>相位灯组界面</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbConflict_2">
       <property name="text">
        <string>冲突相位表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbParttern_2">
       <property name="text">
        <string>方案表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbStage_2">
       <property name="text">
        <string>阶段配时表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbDet_2">
       <property name="text">
        <string>检测器表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbTimeBase_2">
       <property name="text">
        <string>时基表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbSchedule_2">
       <property name="text">
        <string>时段表</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
